trajectory1:
  active: True  # Activates the dynamic obstacle trajectory for testing
  model_name: person_walking_1  # Model has to exist with this name inside the simulation!
  cps:  # At least two points are necessary!
    1: [-9.5,2.5]
    2: [-6.9,2.4]
    3: [-6.0,0.0]
    4: [-6.2,-3.2]
    5: [-6.5,-5.6]
    6: [-3.3,-6.4]
    7: [-3.0,-8.6]
trajectory2:
  active: True
  model_name: person_walking_2
  cps:
    1: [-4.2,6.9]
    2: [-6.4,6.1]
    3: [-6.9,4.5]
    4: [-6.5,2]
    5: [-6.4,-1.0]
    6: [-6.3,-4.0]
    7: [-6.4,-7.0]
    8: [-6.2,-10.0]
    9: [-6.3,-12.0]
    10: [-9.0,-12.7]
trajectory3:
  active: True
  model_name: person_walking_3
  cps:
    1: [-8.8,13.0]
    2: [-10.4,12.6]
    3: [-11.5,11.6]
    4: [-11.7,9.5]
    5: [-11.4,7.2]
    6: [-9.8,6.3]
